website/conferences/views.py

```diff
@@ -241,7 +241,9 @@ def conference_view(**kwargs):
             continue
         meetings.append({
             'name': conf.name,
-            'active': conf.active,
+            'location': conf.location,
+            'end_date': conf.end_date.strftime("%d. %b %Y") if conf.end_date else None,
```

Comment:
@samanehsan After checking multiple conferences' home page, all of them are using the format like `Aug 29, 2015`. Let's do this one!